docs/Project-Configuration/Task-Runtime-Behavior.md

```diff
@@ -257,3 +258,30 @@ Example in a command:
         script: |
           curl -d '{"status":"failed", "type":"setup", "desc":"this should be set", "should_continue": false, "add_failure_metadata_tags": ["failure_tag"]}' -H "Content-Type: application/json" -X POST localhost:2285/task_status
 ```
+
+### Manually Adding Metadata Tags to a Task
+
+The following endpoint was created as a workaround to the above `/task_status`
+agent endpoint for tasks that want to append extra metadata tags to a task after
+the final task status has already been set, such as in the `post`, `timeout`, 
+or `teardown_task` blocks. This endpoint may be called in any command block, and will
+append metadata tags to the existing list of task metadata tags. It can be used in
+conjunction with the `/task_status` endpoint and the `failure_metadata_tags` YAML field.
```

Comment:
Can we also mention that calling this endpoint multiple times allows appending more tags onto the ones that were already appended?